test/clj-tests.lfe

```diff
@@ -515,6 +515,16 @@
         '(2 3 4)      (clj:seq 2 4)
         '(2 4 6 8 10) (clj:seq 2 10 2)))
 
+(deftest conj
+  (is-equal '(1 2 3 4) (clj:conj '(2 3 4) 1))
```

Comment:
These tests with literal expected values can/should use `is-match` instead of `is-equal` so we get better error output should something go wrong.